Woof/Woof/View/DetailedPetSitterView/ContactDelailsSectionView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying a section for the detailed sitter's information.
+struct ContactDelailsSectionView: View {
```

Comment:
typo:
```suggestion
struct ContactDetailsSectionView: View {
```

Reply:
Thanks for your attention! Following suggestion from @NadzeyaShpakouskaya I renamed this element to the  "ReusableLableView" because it's reusable element, where we can provide imageName for an icon and specify the text to show.